Headers/DebugServer2/BreakpointManager.h

```diff
@@ -25,6 +25,12 @@ class BreakpointManager {
     kTypeTemporaryUntilHit = (1 << 2),
   };
 
+  enum Mode : unsigned int {
```

Comment:
What was the reason for the `: unsigned int` here again? I'm sure we talked about it but I don't remember. Maybe you should add a small comment if it's still needed?